### PR TITLE
Add items to portal sample

### PR DESCRIPTION
**Pull request by nanaeaubry**
Update to use new Folders and Folder class

### CODE REVIEW

File: samples/05_content_publishers/publishing_sd_shapefiles_and_csv.ipynb [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "            search_result = gis.content.search(query=file_name, item_type=current_item_type)\n",
```
Comment (cariashuang0417) [inline samples/05_content_publishers/publishing_sd_shapefiles_and_csv.ipynb:65]: <pre class="ql-syntax" spellcheck="false">Line #4.        item = root_folder.add({}, data_dir + current_sd_file)   # .sd file is uploaded and a .sd file item is created
</pre><p>Need "title" and "item_type" for item_properties param, and ".result()" to get the Folder.add result - Same in some of the Folder.add code below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Review by cariashuang0417 — CHANGES_REQUESTED** [review]
Left a comment from nbreviewer above. Just need more info for "title" and "item_type" for item_properties param when doing `Folder.add`, and add `.result()` for getting the results of `Folder.add`

**Review by jyaistMap — CHANGES_REQUESTED** [review]
* submitted #2158 to add navigation and make small changes in formatting and content of dictionary - let me know if you think those don't make sense

**Comment by nanaeaubry** [comment]
@jyaistMap Thank you merged it in :)

**Review by jyaistMap — APPROVED** [review]

**Pull request merged**